py36 unit tests failing

The following unit tests have been failing when you run `tox -e py36`:

```
❯ tox -e py36
GLOB sdist-make: /home/viniarck/repos/kytos-utils/setup.py
py36 recreate: /home/viniarck/repos/kytos-utils/.tox/py36
py36 installdeps: -rrequirements/dev.txt
py36 inst: /home/viniarck/repos/kytos-utils/.tox/.tmp/package/1/kytos-utils-2022.1.zip
py36 installed: astroid==2.9.2,attrs==21.4.0,certifi==2021.10.8,charset-normalizer==2.0.10,click==8.0.3,coverage==6.2,distlib==0.3.4,docopt==0.6.2,filelock==3.4.1,idna==3.3,importlib-met
adata==4.8.3,importlib-resources==5.4.0,iniconfig==1.1.1,isort==5.10.1,Jinja2==2.11.1,kytos-utils==2022.1,lazy-object-proxy==1.7.1,MarkupSafe==1.1.1,mccabe==0.6.1,packaging==21.3,pathspe
c==0.9.0,pep517==0.12.0,pip-tools==6.4.0,platformdirs==2.4.0,pluggy==1.0.0,py==1.11.0,pycodestyle==2.8.0,pylint==2.12.2,pyparsing==3.0.6,pytest==6.2.5,requests==2.27.0,ruamel.yaml==0.17.
19,ruamel.yaml.clib==0.2.6,six==1.16.0,toml==0.10.2,tomli==1.2.3,tox==3.24.5,typed-ast==1.5.1,typing_extensions==4.0.1,urllib3==1.26.7,virtualenv==20.13.0,wrapt==1.13.3,yala==3.1.0,zipp=
=3.6.0
py36 run-test-pre: PYTHONHASHSEED='325290383'
py36 run-test: commands[0] | rm -rf ./python_openflow.egg-info/
py36 run-test: commands[1] | python setup.py ci
running ci
/home/viniarck/repos/kytos-utils/.tox/py36/lib/python3.6/site-packages/setuptools/installer.py:30: SetuptoolsDeprecationWarning: setuptools.installer is deprecated. Requirements should b
e satisfied by a PEP 517 installer.
  SetuptoolsDeprecationWarning,
running coverage
/home/viniarck/repos/kytos-utils/.tox/py36/lib/python3.6/site-packages/setuptools/installer.py:30: SetuptoolsDeprecationWarning: setuptools.installer is deprecated. Requirements should b
e satisfied by a PEP 517 installer.
  SetuptoolsDeprecationWarning,
running pytest
running egg_info
writing kytos_utils.egg-info/PKG-INFO
writing dependency_links to kytos_utils.egg-info/dependency_links.txt
writing requirements to kytos_utils.egg-info/requires.txt
writing top-level names to kytos_utils.egg-info/top_level.txt
reading manifest file 'kytos_utils.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
adding license file 'LICENSE'
adding license file 'AUTHORS.rst'
writing manifest file 'kytos_utils.egg-info/SOURCES.txt'
running build_ext
================================================================================== test session starts ===================================================================================
platform linux -- Python 3.6.15, pytest-6.2.5, py-1.11.0, pluggy-1.0.0
cachedir: .tox/py36/.pytest_cache
rootdir: /home/viniarck/repos/kytos-utils, configfile: setup.cfg
collected 104 items

tests/unit/test_client.py .........                                                                                                                                                [  8%]
tests/unit/test_config.py ....                                                                                                                                                     [ 12%]
tests/unit/test_decorators.py ..                                                                                                                                                   [ 14%]
tests/unit/test_napps.py .............F...F...F.......F.............                                                                                                               [ 55%]
tests/unit/test_openapi.py ...                                                                                                                                                     [ 58%]
tests/unit/test_users.py ...                                                                                                                                                       [ 61%]
tests/unit/commands/test_napps_api.py .........................                                                                                                                    [ 85%]
tests/unit/commands/test_napps_parser.py .......                                                                                                                                   [ 92%]
tests/unit/commands/test_users_api.py .                                                                                                                                            [ 93%]
tests/unit/commands/test_users_parser.py ...                                                                                                                                       [ 96%]
tests/unit/commands/test_web_api.py .                                                                                                                                              [ 97%]
tests/unit/commands/test_web_parser.py ...                                                                                                                                         [100%]

======================================================================================== FAILURES ========================================================================================
_________________________________________________________________________ TestNapps.test_enabled_property__error _________________________________________________________________________

self = <tests.unit.test_napps.TestNapps testMethod=test_enabled_property__error>

    def test_enabled_property__error(self):
        """Test enabled property to error case."""

        with self.assertRaises(SystemExit):
            # pylint: disable=pointless-statement
>           self.napps_manager._enabled
E           AssertionError: SystemExit not raised

tests/unit/test_napps.py:50: AssertionError
______________________________________________________________________ TestNapps.test_get_enabled__connection_error ______________________________________________________________________

self = <tests.unit.test_napps.TestNapps testMethod=test_get_enabled__connection_error>

    def test_get_enabled__connection_error(self):
        """Test method get_enabled to connection error case."""
        with self.assertRaises(KytosException) as context:
>           self.napps_manager.get_enabled()
E           AssertionError: KytosException not raised

tests/unit/test_napps.py:212: AssertionError
_____________________________________________________________________ TestNapps.test_get_installed__connection_error _____________________________________________________________________

self = <tests.unit.test_napps.TestNapps testMethod=test_get_installed__connection_error>

    def test_get_installed__connection_error(self):
        """Test method get_installed to connection error case."""
        with self.assertRaises(KytosException) as context:
>           self.napps_manager.get_installed()
E           AssertionError: KytosException not raised

tests/unit/test_napps.py:180: AssertionError
________________________________________________________________________ TestNapps.test_installed_property__error ________________________________________________________________________

self = <tests.unit.test_napps.TestNapps testMethod=test_installed_property__error>

    def test_installed_property__error(self):
        """Test installed property to error case."""

        with self.assertRaises(SystemExit):
            # pylint: disable=pointless-statement
>           self.napps_manager._installed
E           AssertionError: SystemExit not raised

tests/unit/test_napps.py:69: AssertionError
================================================================================ short test summary info =================================================================================
FAILED tests/unit/test_napps.py::TestNapps::test_enabled_property__error - AssertionError: SystemExit not raised
FAILED tests/unit/test_napps.py::TestNapps::test_get_enabled__connection_error - AssertionError: KytosException not raised
FAILED tests/unit/test_napps.py::TestNapps::test_get_installed__connection_error - AssertionError: KytosException not raised
FAILED tests/unit/test_napps.py::TestNapps::test_installed_property__error - AssertionError: SystemExit not raised
============================================================================= 4 failed, 100 passed in 0.66s ==============================================================================
Command 'coverage3 run setup.py pytest --addopts="tests/unit "&& coverage3 report' returned non-zero exit status 1.
Coverage tests failed. Fix the errors above and try again.
Traceback (most recent call last):
  File "setup.py", line 191, in <module>
    zip_safe=False)
  File "/home/viniarck/repos/kytos-utils/.tox/py36/lib/python3.6/site-packages/setuptools/__init__.py", line 153, in setup
    return distutils.core.setup(**attrs)
  File "/usr/lib/python3.6/distutils/core.py", line 148, in setup
    dist.run_commands()
  File "/usr/lib/python3.6/distutils/dist.py", line 955, in run_commands
    self.run_command(cmd)
  File "/usr/lib/python3.6/distutils/dist.py", line 974, in run_command
    cmd_obj.run()
  File "setup.py", line 137, in run
    check_call(cmd, shell=True)
  File "/usr/lib/python3.6/subprocess.py", line 311, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command 'python3 setup.py coverage --size all --type unit && python3 setup.py lint' returned non-zero exit status 255.
ERROR: InvocationError for command /home/viniarck/repos/kytos-utils/.tox/py36/bin/python setup.py ci (exited with code 1)
________________________________________________________________________________________ summary _________________________________________________________________________________________
ERROR:   py36: commands failed
```

--- a/tests/unit/test_napps.py
+++ b/tests/unit/test_napps.py
@@ -3,6 +3,7 @@ import json
 import re
 import tempfile
 import unittest
+import urllib
 from pathlib import Path, PurePosixPath
 from unittest.mock import MagicMock, Mock, PropertyMock, call, patch
 from urllib.error import HTTPError
@@ -42,9 +43,11 @@ class TestNapps(unittest.TestCase):
 
         self.assertEqual(str(self.napps_manager._enabled), 'ABC')
 
-    def test_enabled_property__error(self):
+    @patch('urllib.request.urlopen')
+    def test_enabled_property__error(self, mock_urlopen):
         """Test enabled property to error case."""
 
+        mock_urlopen.side_effect = urllib.error.URLError("some_error")
         with self.assertRaises(SystemExit):
             # pylint: disable=pointless-statement
             self.napps_manager._enabled
@@ -61,9 +64,11 @@ class TestNapps(unittest.TestCase):
 
         self.assertEqual(str(self.napps_manager._installed), 'DEF')
 
-    def test_installed_property__error(self):
+    @patch('urllib.request.urlopen')
+    def test_installed_property__error(self, mock_urlopen):
         """Test installed property to error case."""
 
+        mock_urlopen.side_effect = urllib.error.URLError("some_error")
         with self.assertRaises(SystemExit):
             # pylint: disable=pointless-statement
             self.napps_manager._installed
@@ -174,8 +179,11 @@ class TestNapps(unittest.TestCase):
         self.assertEqual(installed_napps[0], ("kytos", "mef_eline"))
         self.assertEqual(installed_napps[1], ("kytos", "of_lldp"))
 
-    def test_get_installed__connection_error(self):
+    @patch('urllib.request.urlopen')
+    def test_get_installed__connection_error(self, mock_urlopen):
         """Test method get_installed to connection error case."""
+        err = '[Errno 111] Connection refused'
+        mock_urlopen.side_effect = urllib.error.URLError(err)
         with self.assertRaises(KytosException) as context:
             self.napps_manager.get_installed()
 
@@ -206,8 +214,11 @@ class TestNapps(unittest.TestCase):
         self.assertEqual(installed_napps[0], ("kytos", "mef_eline"))
         self.assertEqual(installed_napps[1], ("kytos", "of_lldp"))
 
-    def test_get_enabled__connection_error(self):
+    @patch('urllib.request.urlopen')
+    def test_get_enabled__connection_error(self, mock_urlopen):
         """Test method get_enabled to connection error case."""
+        err = '[Errno 111] Connection refused'
+        mock_urlopen.side_effect = urllib.error.URLError(err)
         with self.assertRaises(KytosException) as context:
             self.napps_manager.get_enabled()
 

